docs/reference/commandline/stack_config.md

```diff
@@ -26,7 +26,7 @@ Options:
       --help            Print usage
 ```
 
-Displays the configuration of a stack.
+Display the configuration of a stack.
```

Comment:
I strongly prefer active imperative: 

"Use `docker attach` to attach your terminal to a running Docker container.

vs

"The `docker attach` command allows you to attach your terminal to a running Docker container."

It is more concise and easier to read, and the reader is doing the action. In the second way, the command itself is doing the action (allowing). That's not even accurate. The command perhaps "enables" but does not "allow". The reader only cares how to "use"  the command.